examples/Mediawiki/MakeUser.yaml

```diff
@@ -0,0 +1,95 @@
+# This demonstrates the "resource" feature, similar to fixtures.
+# See T228889
+
+resource: User
+description: Creates a user and logs them in
+type: RPC
+
+use-fixtures:
+  - RootUser
+
+variables:
+  name: p/uniq: "User_"
```

Comment:
Would be nice to be able to pass in the wanted name. For example: I want the List Users test to have the same unique prefix.

Reply:
We could allow parameters for resources, I suppose. But I didn't want to complicate things even more.